hopp/simulation/hopp.py

```diff
@@ -33,9 +33,9 @@ def __attrs_post_init__(self) -> None:
         self.system = HybridSimulation(
             site,
             tech_config,
-            self.config.get("dispatch_options"),
-            self.config.get("cost_info"),
-            self.config.get("simulation_options"),
+            self.config.get("dispatch_options") or {},
+            self.config.get("cost_info") or {},
+            self.config.get("simulation_options") or {},
```

Comment:
I was having some headaches managing the type hinting so I made this change, I can change it back if we prefer, but this is overdue for a refactor regardless since it doesn't fit with the code path for the other `HybridSimulation` arguments

Reply:
Ok, yea, let's address this in a future PR then.